core/src/main/java/org/apache/druid/java/util/common/guava/ParallelMergeCombiningSequence.java

```diff
@@ -1036,11 +1047,13 @@ public boolean isReleasable()
     @Override
     public void close()
     {
-      try {
-        yielder.close();
-      }
-      catch (IOException e) {
-        throw new RuntimeException("Failed to close yielder", e);
+      if (yielder != null) {
+        try {
+          yielder.close();
+        }
+        catch (IOException e) {
+          throw new RuntimeException("Failed to close yielder", e);
```

Comment:
I... can't remember if there is a reason why this explodes violently while the others are all `CloseQuietly`... I'll try to figure out if this should be eaten too.

Reply:
It seems the exception will be eventually handled by `CloseQuietly`.